jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/RoadSideUnit.java

```diff
@@ -33,8 +33,22 @@ public static class RSU extends OdeObject {
         public RSU() {
             super();
 
-            // default to 4.1 SNMP protocol
-            this.snmpProtocol = SnmpProtocol.FOURDOT1;
+            String defaultSnmpProtocol = System.getenv("DEFAULT_SNMP_PROTOCOL");
+            if (defaultSnmpProtocol != null) {
+                switch (defaultSnmpProtocol) {
+                case "FOURDOT1":
+                    this.snmpProtocol = SnmpProtocol.FOURDOT1;
+                    break;
+                case "NTCIP1218":
+                    this.snmpProtocol = SnmpProtocol.NTCIP1218;
+                    break;
+                default:
+                    this.snmpProtocol = SnmpProtocol.FOURDOT1;
+                    break;
+                }
+            } else {
+                this.snmpProtocol = SnmpProtocol.FOURDOT1;
+            }

```

Comment:
Rick & I were testing these changes and the defaulting didn't work as expected. We should log the SNMP protocol that was selected and if the specified default is null or unrecognized, an error should be thrown.

Reply:
After further investigation, it seems that the ODE is defaulting to use NTCIP1218 as expected with these changes. I would still like for us to log which SNMP protocol gets selected however. An error being thrown in the default/else cases would also be helpful.

Reply:
Added logging when the default/else cases are reached.